source/services/circle-ci.rst

```diff
@@ -0,0 +1,36 @@
+Circle CI
+=========
+
+The project template now has a Circle CI template that you can use for your project. Circle CI's
+benefit over Travis CI is that it works for private non-Caktus repos. Once you have
+completed the initial project setup (i.e. the items in README.rst), do the following:
+
+1. If you don't have one already, `create a Circle CI account <https://circleci.com/>`_, being sure
+   to link it to your Caktus github account.
+
+#. Find your project on the `projects page <https://circleci.com/add-projects>`_. Any repos that you
+   have admin access to, across organizations, should be visible there. You may need to refresh your
+   list by clicking the 'Reload repositories' button, especially if the repo was created very recently.
+   Enable your repo on this page. This will kick off a build of the default branch of the repo,
+   which will most likely fail, since that branch doesn't yet have a ``circle.yml`` file. Don't
+   worry... we'll fix it.
+
+#. Review the ``circle.yml`` file at the top level of the repo. Update it to match the specifics of
+   your project.
+
+#. Set up HipChat notifications.
+
+   A. Get the Hipchat Room Number. This is the number in the URL bar when you are looking at the
+      room in Hipchat.
+
+   #. Obtain the HipChat API token for Circle CI. Click on the '...' button in the upper right-hand
+      corner of your HipChat room page, then click on 'Integrations...', then click on the
+      'Installed' button to filter by already-installed integrations. You should see Circle CI
+      listed there. Click on it and you will find the Token.
```

Comment:
I *think* I'm an admin, so I added the caveat about possibly needing to create a Caktus tech support request (copied from the travis.rst instructions)